webhooks.go

```diff
@@ -31,3 +34,28 @@ func PostWebhookContext(ctx context.Context, url string, msg *WebhookMessage) er
 func PostWebhookCustomHTTP(url string, httpClient *http.Client, msg *WebhookMessage) error {
 	return PostWebhookCustomHTTPContext(context.Background(), url, httpClient, msg)
 }
+
+func PostWebhookContextCustomHTTP(ctx context.Context, url string, httpClient *http.Client, msg *WebhookMessage) error {
+	return PostWebhookCustomHTTPContext(ctx, url, httpClient, msg)
+}
```

Comment:
Why did you add this function?
To me, `PostWebhookCustomHTTPContext` looks good enough 😄